recipe/meta.yaml

```diff
@@ -23,7 +23,6 @@ requirements:
     - pytest-runner
     - python >=3
   run:
-    - basemap
```

Comment:
Without a build number bump this did not (should bot) create a new package.

Reply:
Sorry, I am still a newbie with conda packaging. How exactly do I do that?

Reply:
Send new PR changing the build number from 0 to 1.